CHANGES.rst

```diff
@@ -727,3 +727,9 @@ Infrastructure, Utility and Other Changes and Additions
 ================
 
 - Initial release.  Includes features!
+
+
+heasarc
```

Comment:
No need for a changelog entry.  We don't add entries for changes that made only to the documentation or tests, or some internal refactors that are invisible to users (rule of thumb is whether it's something that affects users or not). 
I'll remove your second commit before merging, so no need to any follow-ups here.

Reply:
(when a maintainer adds the `no-changelog-entry-needed` label, the check status will get fixed below)